docs/guidance/related_item_guide.rst

```diff
@@ -195,16 +424,16 @@ In this case, the :ref:`12` property is strongly recommended for indexing. In ad
   <relatedItems>
     <relatedItem relationType="IsCitedBy" relatedItemType="JournalArticle">
       <relatedItemIdentifier relatedItemIdentifierType="DOI">10.21384/ExampleJournalArticle</relatedItemIdentifier>
-      <titles>
-        <title>Example Article Title</title>
-      </titles>
       <creators>
         <creator>
           <creatorName nameType="Personal">Miller, Elizabeth</creatorName>
```

Comment:
The 1960 Dissertation by Elizabeth Miller is cited by the 2021 journal article by Elizabeth Miller? So self citation 50 years later? I think it would be much better to have a different person write the article

Reply:
To be clear, these are two separate examples! I can pick a different author to ensure that is unambiguous.

It might help to view the examples in context: https://datacite-metadata-schema.readthedocs.io/en/4.5_relateditem_revision/guidance/related_item_guide.html#use-case-describing-related-resources